### PR TITLE
feat(Table): add "fixed" prop for fixed table layouts

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

File: packages/dnb-eufemia/src/components/table/__tests__/Table.test.tsx [hunk 1]
```diff
@@ -84,15 +84,40 @@ describe('Table', () => {
...
+        <BasicTable />
+      </Table>
+    )
 
     expect(screen.queryByRole('table').getAttribute('aria-label')).toEqual(
       'custom-label'
     )
   })
 
+  it('should set the fixed calss', () => {
```
Comment (thaytharma) [inline packages/dnb-eufemia/src/components/table/__tests__/Table.test.tsx:104]:
```suggestion
  it('should set the fixed class', () => {
```

**Review by langz — APPROVED** [review]

**Review by thaytharma — APPROVED** [review]
Nice!🙌🏽 left a small comment 😊

File: packages/dnb-design-system-portal/src/docs/uilib/components/table/properties.md [hunk 2]
```diff
@@ -10,5 +10,6 @@ showTabs: true
 | `stickyOffset`                              | _(optional)_ defines the offset (top) in `rem` from where the header should start to stick. You may define your app header height here, if you have a sticky header on your page. Defaults to `0`. |
 | ~~`variant`~~ (not implemented yet)         | _(coming)_ defines the style variant of the table. Options: `basis` . Defaults to `generic`.                                                                                                       |
 | ~~`size`~~ (not implemented yet)            | _(coming)_ spacing size inside the table header and data. Options: `small` \| `medium` \| `large` \. Defaults to `large`.                                                                          |
+| `fixed`                                     | _(optional)_ if set to `true`, the table will behave with a fixed table layout, using: `table-layout: fixed;`. Defaults to `null`.                                                                 |
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/table/properties.md:13]: Defaults to `null` or Defaults to `false`?
Technically, as the fixed property of type boolean is not included in defaultProps like fixed: false, I guess it's true that it's value defaults to null 🤔

**Pull request merged**